Tasks/VsTest/vstest.ts

```diff
@@ -187,6 +187,28 @@ function generateResponseFile(): Q.Promise<string> {
     return defer.promise;
 }
 
+function publishCodeChanges(): Q.Promise<string> {
+    var defer = Q.defer<string>();
```

Comment:
as well as abve method

---
In reply to: [69462821](https://github.com/Microsoft/vsts-tasks/pull/2006#discussion_r69462821) [](ancestors = 69462821)